### DISCUSSION TITLE
Ability to name trackers

### DISCUSSION THREAD

**Discussion by purldiver** [seemoo-lab/AirGuard-iOS#4] [Ideas]
I have two trackers for my keys and my bag, and being able to see these in the app is like a QA baseline fame that shows me the app is working. I don’t ignore these trackers because they give me confidence in the app. However, I keep getting alerts about them and they appear the same as all the others in the list. It would be very useful to be able to name them “bag” and “keys”, and turn off caution alerts for them!

**Comment by leonboe1** [comment]
Hi there, as mentioned in the app, trackers change their identity quite often. If you named your trackers, the label would reset at least every day, making this feature impractical to use. The only exception are Tile and Chipolo Trackers, but those can be ignored in the app and then will show up as "safe trackers" in the manual scan.

Best,
Leon
